tests/pfcwd_test.py

```diff
@@ -142,6 +193,67 @@ def test_pfcwd_config_with_ports(self):
         assert result.exit_code == 0
         assert result.output == show_pfcwd_config_with_ports
 
+    def test_pfcwd_start_ports_masic_valid(self):
+        # pfcwd start --action drop --restoration-time 200 Ethernet0 200
+        import pfcwd.main as pfcwd
+        runner = CliRunner()
+        db = Db()
+        # get initial config
+        result = runner.invoke(
+            pfcwd.cli.commands["show"].commands["config"],
+            obj=db
+        )
+        print(result.output)
+        assert result.output == show_pfc_config_all
+
+        result = runner.invoke(
+            pfcwd.cli.commands["start"],
+            [
+                "--action", "forward", "--restoration-time", "101",
+                "Ethernet0", "Ethernet-BP4", "102"
+            ],
+            obj=db
+        )
+        print(result.output)
+        assert result.exit_code == 0
+
+        # get config after the change
+        result = runner.invoke(
+            pfcwd.cli.commands["show"].commands["config"],
+            obj=db
+        )
+        print(result.output)
+        assert result.exit_code == 0
+        assert result.output == show_pfc_config_start_pass
+
+    def test_pfcwd_start_ports_masic_invalid(self):
+        # --action drop --restoration-time 200 Ethernet0 Ethernet500 200
+        import pfcwd.main as pfcwd
+        runner = CliRunner()
+        db = Db()
+
+        result = runner.invoke(
+            pfcwd.cli.commands["start"],
```

Comment:
Isn't the behavior to fail the cmd if something is incorrect? if that's the case, can you add different action and restoration, detection times for Ethernet0 and still the output should be unchanged

Reply:
click handles the options for actions, so will throw an error if invalid option is specified.

Reply:
would be good to add Ethernet0 in this command with diff action and times

Reply:
done